### DISCUSSION TITLE
No commit message field in Theia git panel

### DISCUSSION THREAD

**Discussion by cenkce** [eclipse-theia/theia#9570] [General] [answered]
I build my extension but there is no commit message field in the git panel. Any idea? I also checked using html inspector tool but no message field. And also, without my extension is same.
Theia version : 1.14.0 
and others
``` 
 "@theia/cli": "1.14.0",
    "electron-chromedriver": "9.0.0",
    "app-builder-lib": "22.9.0",
    "concurrently": "^6.2.0",
    "electron": "9.3.2",
    "electron-builder": "^22.8.0"
```
<img width="304" alt="Screen Shot 2021-06-09 at 17 25 26" src="https://user-images.githubusercontent.com/176122/121373947-29bccf00-c948-11eb-9d4c-f211dfdc5995.png">

**Comment by vince-fugnitto** [comment, marked as answer]
@cenkce I could not reproduce the issue from sources (`master`). 

A couple of questions:
- are you using `@theia/git` extension, or the `vscode-builtin-git` plugin for git support?
- are there any errors in the dev-console?
- can you execute the command `reset workbench layout` (resets the layout, perhaps there is something which made the input not render).

**Reply by cenkce** [reply]
Ok I found the problem. theia-scm@1.14.0 > react-autosize-textarea > autosize@4.03 gives an error and they fixed in 4.0.4 and it is working now after update the yarn.lock.json
